lib/shortcuts.js

```diff
@@ -6,20 +6,28 @@ const FileSync = require('lowdb/adapters/FileSync');
 const adapter = new FileSync('shortcuts.json');
 const db = low(adapter);
 
-db.defaults({ shortcuts: [] }).write();
+db.defaults({ users: [] }).write();
 
-const addShortcut = (name, emojis) => {
-  if (db.get('shortcuts').find({ name }).value()) {
+const addShortcut = (userId, name, emojis) => {
+  if (!db.get('users').find({ userId }).value()) {
+    db.get('users').push({ userId: userId, shortcuts: [] }).write();
+  }
+  if (db.get('users').find({ userId }).get('shortcuts').find({ name }).value()) {
     throw new Error(`Shortcut with name ${name} already exists`);
   }
-  db.get('shortcuts').push({ name: name.toLowerCase(), emojis }).write();
+  db.get('users').find({ userId }).get('shortcuts').push({ name: name.toLowerCase(), emojis }).write();
+};
+
+const getAllByUser = (userId) => {
+  return db.get('users').find({ userId }).get('shortcuts').value();
```

Comment:
What if the user does not exists ?

Reply:
It will bugs

Reply:
Not solve yet for me it's not a problem to work on other issues